app/views/assignments/_read.html.erb

```diff
@@ -172,4 +183,20 @@
     <% end %>
   <% end %>
 <% end %>
+  <% unless @grouping.nil? %>
+    <% if allowed_to?(:access_repo?, @grouping) %>
+      <% if @grouping.student_membership_number > 1 %>
+        <h3><%= t('groups.student_interface.url_group_repository') %></h3>
```

Comment:
Let's keep this header as `Repository URL` for both individual and group assignments. (You can remove the unused locale string.)